src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,37 @@
 import React from 'react';
+import { Comments } from '../types/Comments';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comments;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <>
+      <div className="CommentInfo">
+        <div className="CommentInfo__title">
+          <strong className="CommentInfo__name">{name}</strong>
+
+          {' by '}
+
+          <a
+            className="CommentInfo__email"
+            href={`mailto:${email}`}
+          >
+            Telly_Lynch@karl.co.uk
```

Comment:
```suggestion
            Telly_Lynch@karl.co.uk
```
Why you hardcode it?